src/v2/Apps/Search/Components/SearchMeta.tsx

```diff
@@ -1,24 +1,20 @@
-import React, { Component } from "react"
+import React from "react"
 import { Link, Meta, Title } from "react-head"
 import { data as sd } from "sharify"
 
-interface Props {
+interface SearchMetaProps {
   term: string
 }
 
-export class SearchMeta extends Component<Props> {
-  render() {
-    const { term } = this.props
-
-    const title = `Search Results for '${term}' | Artsy`
-    const href = `/search?term=${term}`
-    return (
-      <>
-        <Title>{title}</Title>
-        <Link rel="canonical" href={`${sd.APP_URL}${href}`} />
-        <Meta property="og:title" content={title} />
-        <Meta property="og:url" content={`${sd.APP_URL}${href}`} />
-      </>
-    )
-  }
+export const SearchMeta: React.FC<SearchMetaProps> = ({ term }) => {
+  const title = `Search Results for '${term}' | Artsy`
+  const href = `/search?term=${term}`
+  return (
+    <>
+      <Title>{title}</Title>
+      <Link rel="canonical" href={`${sd.APP_URL}${href}`} />
+      <Meta property="og:title" content={title} />
+      <Meta property="og:url" content={`${sd.APP_URL}${href}`} />
+    </>
+  )
```

Comment:
You can use https://github.com/artsy/force/blob/master/src/v2/Components/MetaTags.tsx here